permissions/plugin-insightappsec.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "insightappsec"
+github: "jenkinsci/insightappsec-plugin"
+paths:
+- "org/jenkins-ci/plugins/insightappsec"
+developers:
+- "ias_developer"
```

Comment:
so you're not allowing `supersloth` to release, I guess on purpose right?

Reply:
So given https://issues.jenkins-ci.org/browse/HOSTING-695?focusedCommentId=358110&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-358110 it does look intended. :+1: